cleverhans/attacks_tf.py

```diff
@@ -57,6 +63,39 @@ def fgsm(x, predictions, eps, clip_min=None, clip_max=None):
     return adv_x
 
 
+def basic_iterative(x, y, model, eps, eps_iter, n_iter,
+                    clip_min=None, clip_max=None):
+    """
+    TensorFlow implementation of the basic iterative method as
+    per Kurakin et al. (2016).
+    Paper link: https://arxiv.org/abs/1607.02533
+    :param x: the input placeholder
+    :param y: the output placeholder
```

Comment:
FYI, in the above mentioned paper I used "hard" labels when computing loss in "Basic iterative" method.
By hard labels I mean that I used 1.0 for true class and 0.0 for all other classes (i.e. no label smoothing).

Kind of the labels does not really change method, and probably both would work well.
But it might be worth noting in the comment, that original paper used hard labels in "basic iterative" method.